src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/ProbationOfficerService.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ProbationOfficerEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ProbationOfficerRepository
+import java.util.UUID
+
+@Service
+class ProbationOfficerService(
+  private val httpAuthService: HttpAuthService,
+  private val probationOfficerRepository: ProbationOfficerRepository
+) {
+  fun getProbationOfficerForRequestUser(): ProbationOfficerEntity {
```

Comment:
Is this something we can use elsewhere, would prevent a lot of calls to the API and ensure we have a source of truth in our database that we use everywhere

Reply:
What else do you think it would be useful for?  I had thought this might potentially be superseded by the outcome of:  https://trello.com/c/0PoVG3S2/710-clarify-source-of-ap-team-data-supervising-teams-officers-and-keyworkers-needed-when-recording-arrivals

Reply:
Yeah, you might be right. I guess going forward there's no point in having two repositories for the same data, especially where things like name etc may change